shared/AppInsightsCore/Tests/Unit/src/W3cTraceParentTests.ts

```diff
@@ -206,5 +209,71 @@ export class W3cTraceParentTests extends AITestClass {
                 } as ITraceParent));
             }
         });
+
+        this.testCase({
+            name: "org generate guid timings",
+            useFakeTimers: false,
+            test: () => {
+                let start = dateNow();
+                for (let lp = 0; lp < 10000; lp++) {
+                    let newId = generateW3CId();
+                }
+
+                let orgTotal = dateNow() - start;
+
+                Assert.ok(orgTotal < 200, "Total time for org generateW3cID = " + JSON.stringify({
```

Comment:
While locally this occurs in around 8ms the build servers are virtualized and slower